<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewData</class>
 <widget class="QDialog" name="ViewData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="twView">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>102</y>
     <width>891</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="tbViewData">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>64</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>View All Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>28</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Data</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="tbUpdateIssue">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>64</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Verify</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbSearch">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>22</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rbCust">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>6</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cust Name</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbCoupon">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>8</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Coupon No</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbDist">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>7</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dist. Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>61</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtSearch">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="tbSearch">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>64</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
